packages/authorization-process/src/services/authorizationService.ts

```diff
@@ -567,6 +570,54 @@ export function authorizationServiceBuilder(
         showUsers: true,
       };
     },
+    async setAdminToClient(
+      {
+        clientId,
+        adminId,
+      }: {
+        clientId: ClientId;
+        adminId: UserId;
+      },
+      { authData, correlationId, logger }: WithLogger<AppContext<UIAuthData>>
+    ): Promise<Client> {
+      assertFeatureFlagEnabled(config, "featureFlagAdminClient");
+
+      await assertUserSelfcareSecurityPrivileges({
+        selfcareId: authData.selfcareId,
+        requesterUserId: authData.userId,
+        consumerId: authData.organizationId,
+        userIdToCheck: adminId,
+        selfcareV2InstitutionClient,
+        correlationId,
+        userRolesToCheck: [userRole.ADMIN_ROLE],
+      });
+
+      logger.info(`Add or update admin in client ${clientId}`);
```

Comment:
```suggestion
      logger.info(`Set user ${adminId} in client ${clientId} as admin`);
```
I think it's best to have this log before `assertUserSelfcareSecurityPrivileges` since that it's already part of its logic